### DISCUSSION TITLE
queryLog to mysql problem

### DISCUSSION THREAD

**Discussion by SLCCL** [0xERR0R/blocky#387] [Q&A] [answered]
Hello guys, wish you all happy holidays.

Back on track, now I'm trying to queryLog to mysql database( I'm using mariadb) for Grafana Dashboard as the documentation.
I'm wondering how to use the syntax of 
"target: db_user:db_password@tcp(db_host_or_ip:3306)/db_user?charset=utf8mb4&parseTime=True&loc=Local" 
on example for Database.
So far on the part of "define the MySQL source in Grafana" is success. The database connection shown "OK" when I click save & test button. But then I go to the dashboard there is show no data. Even I login to the database i created for blocky and  "show tables;", it says it is empty. I would like to know what i've missing or doing wrong. Any help would be appreciated!  (Sorry for my English bad :) )

Thanks again!

**Comment by SLCCL** [comment]
I would like to know should I "pre-create" the table on the database, so that the blocky's queryLog can import data to it? 
Or since my syntax was wrong, so that the blocky's queryLog cannot send data to the database?

**Reply by 0xERR0R** [reply]
The database must exist, blocky will create the database table automatically. Do you see any errors / warnings in your log on startup?

**Comment by SLCCL** [comment, marked as answer]
I think I've figure out what happens, 
"target: db_user:db_password@tcp(db_host_or_ip:3306)/db_user?charset=utf8mb4&parseTime=True&loc=Local"
*second "db_user" change to "db_name" just makes everything works properly.

Thanks.
